pkg/console/subresource/util/util.go

```diff
@@ -54,9 +53,9 @@ func LogYaml(obj runtime.Object) {
 	// deployJSON, err := json.Marshal(d)
 	objYaml, err := yaml.Marshal(obj)
 	if err != nil {
-		logrus.Info("failed to show yaml in log")
+		klog.Info("failed to show yaml in log")
 	}
-	logrus.Infof("%v", string(objYaml))
+	klog.Infof("%v", string(objYaml))
```

Comment:
I know it's not new, but shouldn't this be in part of `else` statement ?

Reply:
agree, will update.